stable/prometheus-operator/templates/exporters/node-exporter/servicemonitor.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app: {{ template "prometheus-operator.name" . }}-node-exporter
 {{ include "prometheus-operator.labels" . | indent 4 }}
 spec:
-  jobLabel: {{ .Values.nodeExporter.jobLabel }}
+  jobLabel: {{ default "node-exporter" .Values.nodeExporter.serviceMonitor.jobLabel }}
```

Comment:
Since you are moving the place where the `jobLabel` is specified for node-exporter this constitutes a breaking change to the chart API - i.e. its values file.

Please bump the major chart version, as per semver requirements: 
>Major version X (X.y.z | X > 0) MUST be incremented if any backwards incompatible changes are introduced to the public API
https://semver.org/#spec-item-8

Please also add migration instructions for someone installing this new chart version to this section of the readme: https://github.com/helm/charts/tree/master/stable/prometheus-operator#upgrading-an-existing-release-to-a-new-major-version